packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barrier-selector.jsx

```diff
@@ -0,0 +1,153 @@
+                        enter: 'trade-container__barriers-table--enter',
+                        enterDone: 'trade-container__barriers-table--enter-done',
+                        exit: 'trade-container__barriers-table--exit',
+                    }}
+                    unmountOnExit
+                >
+                    <Fieldset className='trade-container__fieldset trade-container__barriers-table'>
+                        <div className='trade-container__barriers-table__header'>
+                            <Text color='prominent' weight='bold' size='xs'>
+                                {localize('Barriers')}
+                            </Text>
+                            <div className='trade-container__barriers-table__icon-close' onClick={toggleBarriersTable}>
+                                <Icon icon='IcCross' />
+                            </div>
+                        </div>
+                        <div className='trade-container__barriers-table__text'>Distance to spot</div>
+                        <ThemedScrollbars>
+                            <BarriersList
+                                base_classname='trade-container__barriers-table__item'
+                                active_item_classname='trade-container__barriers-table__item--selected'
+                                className='trade-container__barriers-table__list'
+                                list={turbos_barrier_choices}
+                                selected_item={selected_barrier}
+                                onClick={changeBarrier}
+                                onMouseLeave={onMouseLeave}
+                                onMouseEnter={onMouseEnter}
+                                hover_item_classname='trade-container__barriers-table__item--highlight'
+                            />
+                        </ThemedScrollbars>
+                    </Fieldset>
+                </CSSTransition>
+            )}
+            <Fieldset
+                className='trade-container__fieldset trade-container__barriers'
+                header={localize('Barrier')}
+                is_center
+                header_tooltip={<Localize i18n_default_text={header_tooltip_text} />}
+            >
+                <div onClick={toggleBarriersTable} className='trade-container__barriers__wrapper'>
+                    <div className='trade-container__barriers-spot'>{localize('Spot')}</div>
+                    <InputField
+                        id='dt_barrier_1_input'
+                        type='number'
+                        name='barrier_1'
+                        value={barrier_1}
+                        className='trade-container__barriers-single'
+                        classNameInput={classNames(
+                            'trade-container__input',
+                            'trade-container__barriers-input',
+                            `trade-container__barriers-single-input`
```

Comment:
```suggestion
                            'trade-container__barriers-single-input'
```
@kate-deriv i think we can add 2 barriers support (same as in <Barrier) later